src/IHaskell/IPython.hs

```diff
@@ -53,6 +54,7 @@ data KernelSpecOptions =
 defaultKernelSpecOptions :: KernelSpecOptions
 defaultKernelSpecOptions = KernelSpecOptions
   { kernelSpecGhcLibdir = GHC.Paths.libdir
+  , kernelSpecRTSOptions = []
```

Comment:
Default to _no_ RTS options for the time being (i.e. no memory limit, single processor, GC defaults from GHC.

In 4c1b0597e0381dbd6742543f32e230b2771f4def I propose better (?) defaults.